tests/scripts/deploy_admission_controller.sh

```diff
@@ -3,24 +3,54 @@
 # Sets up the environment for the admission controller webhook in the active cluster.
 set -eo pipefail
 
+trap cleanup SIGINT ERR
+function cleanup() {
+  kubectl -n rook-ceph delete validatingwebhookconfigurations $WEBHOOK_CONFIG_NAME || true
```

Comment:
Actually, I was wrong, you don't need the `|| true`.

Reply:
removed.